test/Microsoft.Health.Fhir.Ingest.Template.UnitTests/IotJsonPathContentTemplateTests.cs

```diff
@@ -35,16 +35,6 @@ public class IotJsonPathContentTemplateTests
             },
         });
 
-        private static readonly IContentTemplate SingleValueMissingTypeNameTemplate = BuildMeasurementExtractor(new IotJsonPathContentTemplate

```

Comment:
Is this just clean up? I am guessing this wasn't being referenced?

Reply:
Correct, in [this comment](https://github.com/microsoft/iomt-fhir/pull/176#discussion_r826460284) of his previous PR, @rogordon01 said he'd remove this.